Formula/sshping.rb

```diff
@@ -0,0 +1,24 @@
+class Sshping < Formula
+  desc "SSH based ping: measure character echo latency and bandwidth"
+  homepage "https://github.com/spook/sshping"
+  url "https://github.com/spook/sshping/archive/refs/tags/v0.1.4.tar.gz"
+  sha256 "589623e3fbe88dc1d423829e821f9d57f09aef0d9a2f04b7740b50909217863a"
+  license "MIT"
+
+  depends_on "cmake" => :build
+  depends_on "pod2man" => :build
+  depends_on "libssh"
+
+  def install
+    rm "Makefile"
+    system "cmake", *std_cmake_args
+    system "make"
+    system "make", "install"
+    man8.install "doc/sshping.8"
+  end
+
+  test do
+    assert_equal "Usage: sshping [options] [user@]addr[:port]",
+      shell_output("#{bin}/sshping -h 2>&1 | head -n 1").strip
```

Comment:
We need a test that exercises the some of the functionality of the app. Version checks or usage checks (foo --version or foo --help) are not sufficient, as explained in the formula cookbook.

In most cases, a good test would involve running a simple test case: run #{bin}/foo input.txt.

- Then you can check that the output is as expected (with assert_equal or assert_match on the output of shell_output)
- You can also check that an output file was created, if that is expected: assert_predicate testpath/"output.txt", :exist?

Some advice for specific cases:

- If the formula is a library, compile and run some simple code that links against it. It could be taken from upstream's documentation / source examples.
- If the formula is for a GUI program, try to find some function that runs as command-line only, like a format conversion, reading or displaying a config file, etc.
- If the software cannot function without credentials, a test could be to try to connect with invalid credentials (or without credentials) and confirm that it fails as expected.
- Same if the software requires a virtual machine, docker instance, etc. to be running.